SOURCEBUILD.md

```diff
@@ -144,6 +144,30 @@ This project uses cmake for building the C/C++ library. To configure cmake, run:
   ```
 The default installation locations are `$CMAKE_INSTALL_PREFIX/lib` and `$CMAKE_INSTALL_PREFIX/include/cugraph` respectively.
 
+#### Updating the RAFT branch
+
+`libcugraph` uses the [RAFT](https://github.com/rapidsai/raft) library and there are times when it might be desirable to build against a different RAFT branch, such as when working on new features that might span both RAFT and cuGraph. 
+
+For local development, the `CPM_raft_SOURCE=<path/to/raft/source>` option can be passed to the `cmake` command to enable `libcugraph` to use the local RAFT branch.
```

Comment:
We really have two approaches to this.

1. CPM_raft_SOURCE which means for raft changes to be seen by cugraph you just need to build cugraph.  
2. CMAKE_PREFIX_PATH to the raft build directory.  This will allow you to use an already built version of raft, but any modifications of raft will require you to rebuild it ( unless the changes are header only )